framework/src/controller/network/network.js

```diff
@@ -347,50 +311,63 @@ module.exports = class Network {
 		}
 	}
 
-	get actions() {
-		return {
-			request: async action =>
-				this.p2p.request({
-					procedure: action.params.procedure,
-					data: action.params.data,
-				}),
-			send: action =>
-				this.p2p.send({
-					event: action.params.event,
-					data: action.params.data,
-				}),
-			requestFromPeer: async action =>
-				this.p2p.requestFromPeer(
-					{
-						procedure: action.params.procedure,
-						data: action.params.data,
-					},
-					action.params.peerId,
-				),
-			sendToPeer: action =>
-				this.p2p.sendToPeer(
-					{
-						event: action.params.event,
-						data: action.params.data,
-					},
-					action.params.peerId,
-				),
-			broadcast: action =>
-				this.p2p.broadcast({
-					event: action.params.event,
-					data: action.params.data,
-				}),
-			getConnectedPeers: () => this.p2p.getConnectedPeers(),
-			getDisconnectedPeers: () => this.p2p.getDisconnectedPeers(),
-			applyPenalty: action =>
-				this.p2p.applyPenalty({
-					peerId: action.params.peerId,
-					penalty: action.params.penalty,
-				}),
-		};
+	async request(requestPacket) {
+		return this.p2p.request({
+			procedure: requestPacket.procedure,
+			data: requestPacket.data,
+		});
+	}
+
+	send(sendPacket) {
+		return this.p2p.send({
+			event: sendPacket.event,
+			data: sendPacket.data,
+		});
+	}
+
+	async requestFromPeer(requestPacket) {
+		return this.p2p.requestFromPeer(
+			{
+				procedure: requestPacket.procedure,
+				data: requestPacket.data,
+			},
+			requestPacket.peerId,
+		);
+	}
+
+	sendToPeer(sendPacket) {
+		return this.p2p.sendToPeer(
+			{
+				event: sendPacket.event,
+				data: sendPacket.data,
+			},
+			sendPacket.peerId,
+		);
+	}
+
+	broadcast(broadcastPacket) {
+		return this.p2p.broadcast({
+			event: broadcastPacket.event,
+			data: broadcastPacket.data,
+		});
+	}
+
+	getConnectedPeers() {
+		return this.p2p.getConnectedPeers();
+	}
+
+	getDisconnectedPeers() {
+		return this.p2p.getDisconnectedPeers();
+	}
+
+	applyPenalty(penaltyPacket) {
+		return this.p2p.applyPenalty({
+			peerId: penaltyPacket.peerId,
+			penalty: penaltyPacket.penalty,
+		});
 	}
 
-	async cleanup() {
+	stopNetwork() {
```

Comment:
Having `stop` in the `Network` class is good. 

```suggestion
	stop() {
```